From be06ab3ee7266c6ca7d2475d45b42cc4f47a20f8 Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bossart <pierre-louis.bossart@intel.com>
Date: Thu, 19 Aug 2010 20:42:30 -0500
Subject: [PATCH] HDA-Intel: do not lock IEC958 Playback switch

As discussed with Takashi, removing the lock allows apps to mute
the output using the mixer interface.
Other AES controls remain locked.

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@intel.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 src/conf/cards/HDA-Intel.conf |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/src/conf/cards/HDA-Intel.conf b/src/conf/cards/HDA-Intel.conf
index 726e8c9..f7eecb4 100644
--- a/src/conf/cards/HDA-Intel.conf
+++ b/src/conf/cards/HDA-Intel.conf
@@ -111,8 +111,6 @@ HDA-Intel.pcm.iec958.0 {
 			}
 			{
 				name "IEC958 Playback Switch"
-				lock true
-				preserve true
 				value true
 			}
 			]
@@ -184,8 +182,6 @@ HDA-Intel.pcm.hdmi.common {
 		{
 			name "IEC958 Playback Switch"
 			index $CTLINDEX
-			lock true
-			preserve true
 			value true
 		}
 		]
-- 
1.7.6

